ui/app/controllers/vault/cluster/access/oidc/clients/client.js

```diff
@@ -9,6 +9,7 @@ export default class OidcClientController extends Controller {
   constructor() {
     super(...arguments);
     this.router.on('routeDidChange', ({ targetName }) => {
+      console.log(targetName);
```

Comment:
do we intentionally want this console log in here?

Reply:
sure don't! thank you